.github/workflows/rebuildDependencies.yml

```diff
@@ -0,0 +1,78 @@
+name: VCMI - vcpkg dependencies
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 2 * * 0' #every sunday 02:00
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - platform: msvc-x64
+            triplet: x64-windows
+          - platform: msvc-x86
+            triplet: x86-windows
+
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout vcpkg repository
+      uses: actions/checkout@v4
+      with:
+        repository: 'microsoft/vcpkg'
```

Comment:
`--depth 1` should clone it faster

Reply:
Looks like it is already the case:
https://github.com/actions/checkout
```
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```